src/main/java/uk/gov/hmcts/reform/mi/miextractionservice/service/impl/BlobExportServiceImpl.java

```diff
@@ -51,19 +62,55 @@ public void exportBlobs() {
         OffsetDateTime toDate  = StringUtils.isEmpty(retrieveToDate)
             ? getEndOfDay(dateTimeUtil.getCurrentDateTime().minusDays(1L)) : getEndOfDay(dateTimeUtil.parseDateString(retrieveToDate));
 
+        BlobServiceClient stagingClient = extractionBlobServiceClientFactory.getStagingClient();
         BlobServiceClient exportClient = extractionBlobServiceClientFactory.getExportClient();
 
-        // Export CCD Blobs
-        String outputBlobName = ccdExportBlobDataComponent.exportBlobsAndGetOutputName(
-            extractionBlobServiceClientFactory.getStagingClient(),
+        List<String> messages = new ArrayList<>();
+
+        if (dataSource.equalsIgnoreCase(NO_FILTER_VALUE) || dataSource.equalsIgnoreCase(SourceEnum.CORE_CASE_DATA.getValue())) {
```

Comment:
not for now, but we probably need to think something for the future to make this more configuration based. 
we don't want to end up with 10s if dataSource.equasIgnoreCase...

Reply:
Makes sense, I'll probably do some improvements across the repos before the payment ingestion starts.